src/backend/columnar/sql/citus_columnar--11.1-1--11.2-1.sql

```diff
@@ -0,0 +1,32 @@
+-- add columnar objects back
+ALTER EXTENSION citus_columnar ADD SCHEMA columnar;
+ALTER EXTENSION citus_columnar ADD SCHEMA columnar_internal;
```

Comment:
we should drop those lines because those changes are already done in the prev migration file